### PR TITLE
AccuWeather (Independent Connector)

### PR DESCRIPTION
**Pull request by ahmadn82**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

image
image
image

image

### CODE REVIEW

File: independent-publisher-connectors/AccuWeather/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          "tooltip": "Provide API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FF4400",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/AccuWeather/apiProperties.json:18]: @ahmadn82 Please update it with `"#da3b01"` as it is the default iconBrandColor for all the Independent Publisher connectors.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/AccuWeather/apiProperties.json (hunk 1 above)
Comment (ahmadn82) [reply independent-publisher-connectors/AccuWeather/apiProperties.json:18]: Fixed! @sriyen-msft

File: independent-publisher-connectors/AccuWeather/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,7900 @@
...
+              "items": {
+                "type": "object",
+                "properties": {
+                  "ID": {
+                    "type": "string",
+                    "description": "ID"
+                  },
+                  "LocalizedName": {
+                    "type": "string",
+                    "description": "LocalizedName"
```
[the remark this answers is not in the document]
Comment (ahmadn82) [reply independent-publisher-connectors/AccuWeather/apiDefinition.swagger.json:36]: Done! @sriyen-msft

**Review by sriyen-msft — APPROVED** [review]

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@ahmadn82 We had issues while setting up API for your connector, which caused slight delay production deployment of your connector. We will try to fastrack the deployment and keep you posted. Thanks.

**Comment by ahmadn82** [comment]
> @ahmadn82 We had issues while setting up API for your connector, which caused slight delay production deployment of your connector. We will try to fastrack the deployment and keep you posted. Thanks.

@sriyen-msft Let me know if there is anything I can do to help 😃

**Comment by sriyen-msft** [comment]
> > @ahmadn82 We had issues while setting up API for your connector, which caused slight delay production deployment of your connector. We will try to fastrack the deployment and keep you posted. Thanks.
> 
> @sriyen-msft Let me know if there is anything I can do to help 😃

Thank you so much for your patience, Your connector is started rolling out to Production regions from today, and it is expected to complete production deployment on or before 20th September 2021. Please let me know if you any questions. Thanks

**Comment by natalie-pienkowska** [comment]
Congrats, Ahmad :)

**Comment by ahmadn82** [comment]
> > > @ahmadn82 We had issues while setting up API for your connector, which caused slight delay production deployment of your connector. We will try to fastrack the deployment and keep you posted. Thanks.
> > 
> > 
> > @sriyen-msft Let me know if there is anything I can do to help 😃
> 
> Thank you so much for your patience, Your connector is started rolling out to Production regions from today, and it is expected to complete production deployment on or before 20th September 2021. Please let me know if you any questions. Thanks

Thank you guys! ❤️

**Comment by ahmadn82** [comment]
> Congrats, Ahmad :)

Thank you @natalie-pienkowska

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**